internal/controller/datadogagent/feature/otlp/feature_test.go

```diff
@@ -126,8 +293,9 @@ func TestOTLPFeature(t *testing.T) {
 		{
 			Name: "[single container] gRPC enabled, no APM",
 			DDA: newAgentSingleContainer(Settings{
-				EnabledGRPC:  true,
-				EndpointGRPC: "0.0.0.0:4317",
+				EnabledGRPC:         true,
+				EnabledGRPCHostPort: true,
```

Comment:
The config setup in `builder.go` means that this test is setting the grpc host port to `0` instead of not setting the port value at all, which I believe is the intent